tableau_api.gemspec

```diff
@@ -19,19 +17,20 @@ Gem::Specification.new do |spec|
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
-  spec.required_ruby_version = '~> 2.1'
+  spec.required_ruby_version = '>= 2.1'
```

Comment:
Would like to hear some thoughts on bumping the required ruby version, or if 2.1 should be added to the list of travis versions!

Reply:
If it works with 2.1 then I say add it to the Travis version list. Maybe @vfu116 has other opinions, though?

Reply:
Leaving 2.1 seems fine to me but why not just bump to 2.7?